LocalSettings.php

```diff
@@ -1354,6 +1354,11 @@
 	'wmgUseWikibaseRepository' => [
 		'default' => false,
 	],
+	// Do not put in ManageWiki
```

Comment:
Why?

Reply:
Because of how it's configured see https://www.mediawiki.org/wiki/Wikibase/Installation/Advanced_configuration#Wikibase_Client